orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/TemplateMerge.java

```diff
@@ -89,7 +89,13 @@ private static void mergeConfiguration(PipelineTemplate a, PipelineTemplate b) {
       boolean updated = false;
       for (int i = 0; i < merged.size(); i++) {
         if (merged.get(i).getName().equals(bNode.getName())) {
-          merged.set(i, bNode);
+          if (bNode.isRemove()) {
```

Comment:
I see how this handles inheritance between parent and child templates, but does this bit also handle merge/remove as defined by the template config? Or am I missing something below?